.gitlab-ci.yml

```
@@ -0,0 +1,44 @@
+    - docker push $DOCKERHUB_USER/notes-app:latest
+    - echo "Pushing image to dockerhub....."
+  tags:
+    - dev
+  artifacts:
+    paths:
+      -  builds/
+    expire_in: 1 day
+
+deploy_job:
+  stage: deploy_on_EC2
+  script: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)